department-of-veterans-affairs/va.gov-cms
# VACMS-4085: Improve revisions and proofing for facility nodes.

## Description

See #4085. 

This depends on #2265 (PR #4086) because of the string_field_formatter module 

## Testing done

Manual 
## Screenshots


## QA steps

1. Go to /node/84/revisions/view/14396/360419/visual_inline
2. You should be able to see diffs of "Prepare for your visit" (compare to screenshot on #4085 to see improvement) 



### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


until #4086 is merged.

### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
